src/porepy/numerics/fv/fvutils.py

```diff
@@ -188,9 +186,11 @@ def compute_dist_face_cell(
 
     _, cols = np.meshgrid(subcell_topology.subhfno, np.arange(dims))
     cols += matrix_compression.rldecode(np.cumsum(blocksz) - blocksz[0], blocksz)
-    eta_vec = eta * np.ones(subcell_topology.fno.size)
-    # Set eta values to zero at the boundary
-    if not eta_at_bnd:
+    if np.asarray(eta).size == subcell_topology.num_subfno_unique:
```

Comment:
Is eta interpreted according to its length? Documentation is needed at the function level.